.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: housekeeping
+on: [push, pull_request]
+jobs:
+    tsc-and-format:
+        runs-on: ubuntu-latest
```

Comment:
Below `runs-on` you should add a custom timeout, because the gha default is one hour and if the build fails from a mysterious reason it won't fail quickly enough
```yaml
timeout-minutes: 10
```

Reply:
Added this in 1cbf73c